tre_schemas/avro/tre-judgment-available.avsc

```diff
@@ -54,33 +54,33 @@
         "type": "record",
         "name": "Parameters",
         "fields": [
-          {
-            "name": "s3FolderUrl",
+         {
+            "name": "reference",
             "type": "string",
-            "description": "s3-folder-url"
+            "description": "(TDR-2023-AAA|FCL-???????) # opaque string: use originator to know where it came from, should be an acceptable filename"
           },
           {
-            "name": "judgmentDocx",
-            "type": "string",
-            "description": "docx file path and name"
+               "name": "originator",
+               "type": "string",
+               "description":" who provided the docx FCLTDR)"
           },
           {
-             "name": "judgmentXml",
-             "type": "string",
-             "description": "xml file path and name"
-           },
-           {
-              "name": "parserLog",
+            "name": "bundleFileURI",
+            "type": ["null", "string"],
+            "description": "docx URI; may be absent if request was from FCL"
+          },
```

Comment:
To my understanding this field is where putting our AWS s3 link for FCL to use (as per example above) so I think the description needs tweaking.

Reply:
schema updated